conda-recipes/versions.yaml

```diff
@@ -1,6 +1,6 @@
-version: 4.4.0
+version: 4.5.0
```

Comment:
Same comment as in main repo. How about 4.4.1 as a patch release?

Reply:
Done